src/main/java/com/cgi/ipsen5/Controller/UserController.java

```diff
@@ -43,6 +47,19 @@ public ApiResponse<UserResponseDTO> getUserById(@PathVariable UUID id) {
         }
     }
 
+    @GetMapping(path = "/{id}/get-favorite-colleagues")
+    public ApiResponse<List<UserFavoriteColleaguesDTO>> getFavoriteColleaguesByUserId(@PathVariable UUID id){
+        List<UserFavoriteColleaguesDTO> favorites;
+        favorites = this.favoriteColleagueService.getFavoriteColleaguesFromEmployee(id);
+        return new ApiResponse<>(favorites);
```

Comment:
je kan hier beter het volgende doen:
return new ApiResponse<>(this.favoriteColleagueService.getFavoriteColleaguesFromEmployee(id), en dan hier status);